app/code/Magento/Catalog/Model/Config.php

```diff
@@ -395,7 +411,7 @@ public function getProductTypeName($id)
      * @param string $value
      * @return null|mixed
      */
-    public function getSourceOptionId($source, $value)
+    public function getSourceOptionId($source, string $value)
```

Comment:
@Usik2203 lets revert this change.
```suggestion
    public function getSourceOptionId($source, $value)
```